projects/05-fleet-management-api/project.yml

```
@@ -1,10 +1,43 @@
 track: web-dev
 learningObjectives:
-  - java
-  - python
-  - object-oriented-programming
-  - node
   - sql
   - db
   - postgres/data-types
   - postgres/indexes
+
+# permite mayor control de la informacion en seleccion de proyecto
+# se pretende tener una semantica predefinida q controle el comportamiento
+# de la interfaz.
+# por ahora estos son los unicos valores permitidos
+tags:
+  - featured
+  - beta
+  - deprecated
+  - hidden
+
+# para proyectos que tienen diferentes implementaciones (aka rutas)
+variants:
+  - name: java
+    learningObjectives:
+      - java
+      - object-oriented-programming
+  - name: node
+    learningObjectives:
+      - node
+      # Cualquier OA puede expresarse como objeto, por ejemplo para flaggear
+      # como "opcional".
+      # por ahora es la unica propiedad soportada.
+      - object-oriented-programming:
+          optional: true
+  - name: python
+    learningObjectives:
+      - python
+      - object-oriented-programming:
+          optional: true
+  - name: python-django
+    learningObjectives:
+      - python
+      - django      
```

```suggestion
      - python/django
```

Los OAs deberían seguir escribiéndose como se escriben en los demás proyectos, para saber que esa propiedad corresponde a los OAs de `python`, o si no el parser creerá que ese OA (`django` por sí solo) no existe, y por ende arroja error (`Unknown learning objectives: django.`); Este comportamiento es así porque (quizás con la palabra _django_ no ocurrirá, pero) existen OAs "repetidos", por ejemplo `functions` es tanto un OA _dentro_ del OA `js` como de `csharp`, `php`, `python`, etc... Y algunos incluyen cosas que otros no.

Entonces mejor mantener la convención :blush: